tools/pkg/build.py

```diff
@@ -0,0 +1,356 @@
+    with zipfile.ZipFile(onedir_artifact, mode="r") as archive:
+        archive.extractall(unzip_dir)
+
+    move_dir = unzip_dir / "salt"
+    build_env = unzip_dir / "buildenv"
+    _check_pkg_build_files_exist(ctx, move_dir=move_dir)
+
+    ctx.info(f"Moving {move_dir} directory to the build environment in {build_env}")
+    shutil.move(move_dir, build_env)
+
+    ctx.info("Building the windows package")
+    ctx.run(
+        "powershell.exe",
+        "&",
+        "pkg/windows/build.cmd",
+        "-Architecture",
+        arch,
+        "-Version",
+        salt_version,
+        "-CICD",
+        "-SkipInstall",
+    )
+
+    ctx.info("Done")
+
+
+@build.command(
+    name="onedir-dependencies",
+    arguments={
+        "arch": {
+            "help": "The architecture to build the package for",
+            "choices": ("x86_64", "aarch64", "x86", "amd64"),
+            "required": True,
+        },
+        "python_version": {
+            "help": "The version of python to create an environment for using relenv",
+            "required": True,
+        },
+        "package_name": {
+            "help": "The name of the relenv environment to be created under artifacts/",
+            "required": True,
+        },
+        "platform": {
+            "help": "The platform the relenv environment is being created on",
+            "required": True,
+        },
+    },
+)
+def onedir_dependencies(
+    ctx: Context,
+    arch: str = None,
+    python_version: str = None,
+    package_name: str = None,
+    platform: str = None,
+):
+    """
+    Create a relenv environment with the onedir dependencies installed.
+
+    NOTE: relenv needs to be installed into your environment and builds and toolchains (linux) fetched.
+    """
+    if TYPE_CHECKING:
+        assert arch is not None
+        assert python_version is not None
+        assert package_name is not None
+        assert platform is not None
+
+    # We import relenv here because it is not a hard requirement for the rest of the tools commands
+    try:
+        from relenv.create import create
+    except ImportError:
+        ctx.exit(1, "Relenv not installed in the current environment.")
+
+    artifacts_dir = pathlib.Path("artifacts").resolve()
+    artifacts_dir.mkdir(exist_ok=True)
+    dest = artifacts_dir / package_name
+
+    create(dest, arch=arch, version=python_version)
+
+    if platform == "windows":
+        python_bin = dest / "Scripts" / "python"
+        no_binary = []
+    else:
+        python_bin = dest / "bin" / "python3"
+        no_binary = ["--no-binary=:all:"]
+
+    version_info = ctx.run(
+        str(python_bin),
+        "-c",
+        "import sys; print('{}.{}'.format(*sys.version_info))",
+        capture=True,
+    )
+    requirements_version = version_info.stdout.strip().decode()
+    requirements_file = pathlib.Path(
+        "requirements", "static", "pkg", f"py{requirements_version}", f"{platform}.txt"
+    ).resolve()
```

Comment:
```suggestion
    requirements_file = tools.utils.REPO_ROOT / "requirements" / "static" / "pkg" / f"py{requirements_version}" / f"{platform}.txt"
```
FYI, `tools.utils.REPO_ROOT` exists